tests/unit/test_proxy.py

```diff
@@ -32,7 +32,6 @@ def test_start_and_stop(self, monkeypatch):
             backend_port,
             update_listener=None,
             quiet=True,
-            params={"protocol_version": "HTTP/1.0"},
```

Comment:
Value hasn't been used/processed downstream, so this change does not change the logic itself.